content/docs/instrumenting/exporters.md

```diff
@@ -58,6 +58,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Twemproxy](https://github.com/stuartnelson3/twemproxy_exporter)
 
 ### Hardware related
+   * [A10 exporter](https://github.com/a10networks/PrometheusExporter)
```

Comment:
There's no need for this entry, it's only for older software and the link below would presumably point users in the right direction.